deploy/docker/seqr/Dockerfile

```diff
@@ -52,38 +68,44 @@ RUN wget https://raw.github.com/miyagawa/cpanminus/master/cpanm -O /usr/bin/cpan
 # install dev dependencies for react, javascript development. These are not needed at runtime.
 RUN apt-get update \
     && curl -sL https://deb.nodesource.com/setup_8.x | bash - \
-    && apt-get install -y \
-        nodejs
+    && apt-get install -y --no-install-recommends \
+        nodejs \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
 
-# install database clients for debugging
-RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ jessie-pgdg main' >> /etc/apt/sources.list.d/postgresql.list \
-    && wget --no-check-certificate -q https://www.postgresql.org/media/keys/ACCC4CF8.asc -O- | apt-key add - \
-    && apt-get update && apt-get install -y \
-        postgresql \
-        postgresql-client \
-        libpq-dev
+# install database clients for debugging (https://www.postgresql.org/download/linux/debian/)
+RUN echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list \
```

Comment:
Codacy found an issue: [Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`](https://app.codacy.com/manual/hanars/seqr/pullRequest?prid=5891012)